modules/ossm-rn-fixed-issues.adoc

```diff
@@ -32,6 +32,8 @@ $ oc delete crd orders.certmanager.k8s.io
 $ oc delete crd challenges.certmanager.k8s.io
 ----
 
+* link:https://issues.redhat.com/browse/TRACING-1300[TRACING-1300] Failed connection between agent and collector when using Istio sidecar. An update of the Jaeger Operator in spring enabled TLS communication by default between a Jaeger sidecar agent and the Jaeger collector.
```

Comment:
I would remove "in spring" - not sure what this relates to.

Reply:
@objectiser Good point. Removed.